CSAR.lua

```diff
@@ -306,7 +249,7 @@ function csar.eventHandler:onEvent(_event)
                         --env.info(string.format("Schedule Respawn %s %s",_heliName,_woundedName))
                         -- queue up script
                         -- Schedule timer to check when to pop smoke
-                        timer.scheduleFunction(csar.checkWoundedGroupStatus, { _heliName, _woundedName }, timer.getTime() + 5)
+                 --       timer.scheduleFunction(csar.checkWoundedGroupStatus, { _heliName, _woundedName }, timer.getTime() + 5)
```

Comment:
Why did you remove this? Been a while since I looked but think you need this as an alternative to the f10? maybe add an option to pick between automated and f10?

Reply:
you're good..  I didn't remember why I removed it from there..  but it had to be there.  I added the schedule when Insert the unit with prefix in the function below, but if the player gets out and get back to the same unit for sure the will be some issues.  So I added it back in the latest commit.